tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -8361,7 +8361,45 @@ TEST_F(ExecutionTest, DynamicResourcesTest) {
     return;
   }
 
-  RunResourceTest(pDevice, pShader, L"cs_6_6", /*isDynamic*/true);
+  RunResourceTest(pDevice, pShader, L"cs_6_6", /*isDynamic*/false);
```

Comment:
No reason to change this to false. Probably a temp change that didn't get reverted